bin/getdata.js

```diff
@@ -1,112 +1,104 @@
 #!/usr/bin/env node --harmony
 
-var program = require('commander');
-var co = require('co');
-var prompt = require('co-prompt');
-var request = require('request');
-var fs = require('fs');
-var chalk = require('chalk');
+/* eslint-disable no-console */
 
-program
-	.version('0.0.1')
-	.arguments('<authemail> <id>')
-	.option('-p, --password <password>',
-		'Password for authemail. Recommended flag for piping to another tool.')
-	.option('-o, --output <output>', 'path/to/output.json')
-	.option('--dev', 'Use development server. Default server is production.')
-	.option('--stg', 'Use staging server. Default server is production.')
-	.option('--int', 'Use integration server. Default server is production.')
-	.option('--clinic', 'Use clinic server. Default server is production.')
-	.option('-v, --verbose', 'Verbose output.')
-	.action(function (authemail, id) {
-
-		var password;
-		var session_token;
-		co(function* () {
-				if (program.password)
-					password = program.password;
-				else
-					password = yield prompt.password('Password: ');
-			})
-
-			.then(function () {
-				var env = getEnvironment();
-
-				var url = makeLoginRequestUrl(authemail, password, env);
-				request.post({
-						url: url
-					},
-					function (error, response, body) {
-
-						exitOnError(error,
-							response ? response.statusCode : null,
-							'An error occurred with login request. ' +
-							'Bad username/password? ');
-
-						if (program.verbose) {
-							console.log(chalk.green.bold('Successful login.'));
-						}
-
-						var req = makeDataQueryRequest(id,
-							response
-							.headers['x-tidepool-session-token'],
-							env);
-
-						var outstream = makeOutstream(program.output);
-
-						request.get(req, function (error, response, body) {
-
-							exitOnError(error, response.statusCode,
-								'An error occurred with data request. ' +
-								'Incorrect id for data? ');
-
-							if (program.verbose) {
-								console.log(chalk.green.bold('Successful data request.'));
-							}
-						}).pipe(outstream);
-
-					});
-			});
-
-	})
-	.parse(process.argv);
+const program = require('commander');
+const prompt = require('prompt-sync')();
+const rp = require('request-promise-native');
```

Comment:
`rp`  maybe just `requestPromise` for readability? I only say that as I had to scroll to the top to figure out what it was